### DISCUSSION TITLE
[V3] Optimize Blur to only send network requests when the field is changed

### DISCUSSION THREAD

**Discussion by twellck** [livewire/livewire#6809] [4. General]
Working with Livewire v3's blur has been great, I have noticed that network requests are being sent even if the field(s) with `wire:model.blur` have not been updated (value). 

This is quite expensive especially if a user is tabbing through a form. 

While this is not necessarily an unexpected behaviour, I cannot not think of a reason to send those requests if the field value has not changed.

_I'd be happy to create an issue / PR but would prefer to discuss whether this change on the `blur` modifier would make sense for everyone or if some extra modifier would fit better._

**Comment by mitsyara** [comment]
agree!

I had a Filament table with over 1M records. Whenever the user tabbed through the page, the network request was triggered, although nothing really changed. Very expensive.
